commercetools-models/src/test/java/io/sphere/sdk/carts/CartLikeTest.java

```diff
@@ -0,0 +1,51 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.sphere.sdk.models.DefaultCurrencyUnits.EUR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CartLikeTest {
+
+    private Cart cartWithTaxes;
+    private Cart cartWithoutTaxes;
+
+    @Before
+    public void setUp() throws Exception {
+        cartWithTaxes = SphereJsonUtils.readObjectFromResource("cart-with-taxes.json", Cart.class);
+        cartWithoutTaxes = SphereJsonUtils.readObjectFromResource("cart-without-taxes.json", Cart.class);
+    }
+
```

Comment:
Is there any reason not to use a mocking framework?

Can you use the `*Impl` for the tests?
Maybe you could load a "base" cart with json and set the fields relevant for the tests in java.
(you may have to use `instanceof` and `*Impl` classes for that - not particularly pretty)

Otherwise I'd go with minimizing the json files.